src/main/java/com/sngular/kloadgen/sampler/schemaregistry/impl/ApicurioSchemaRegistry.java

```diff
@@ -153,7 +151,7 @@ private SearchedArtifact getLastestSearchedArtifact(final String subjectName) {
     boolean found = false;
     SearchedArtifact best = null;
     final Comparator<SearchedArtifact> comparator = Comparator.comparing(SearchedArtifact::getModifiedOn);
-    for (SearchedArtifact artifact : this.schemaRegistryClient.searchArtifacts(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_DEFAULT_GROUP, null, null, null, null, null, null, null,
+    for (SearchedArtifact artifact : this.schemaRegistryClient.searchArtifacts(null, null, null, null, null, null, null, null,
                                                                                null,
                                                                                null, null).getArtifacts()) {
```

Comment:
This method make sense with all the value as null? Maybe a new method could be a good idea? Let me know what you think about that

Reply:
It seems strange, but io.apicurio.registry.rest.client.RegistryClient doest not provide a generic method to retrieve all artifacts registered. This looks the "best" approach

Reply:
Ok!